### PR TITLE
[Improvement] Unify the builder of VOs via lombok.Builder

### PR DESCRIPTION
**Pull request by s7monk**
### Purpose

The builder of VOs could be unified via `lombok.Builder`. Meanwhile fix the problems of `MetadataControllerTest`.

### Tests

- `MetadataControllerTest`

### API and Format

No.

### Documentation

No.

### CODE REVIEW

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 1]
```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
                                                                 new TypeToken<
                                                                         LinkedList<
                                                                                 MetadataFieldsModel>>() {}))
-                                        .setPartitionKeys(internalRow.getString(2).toString())
-                                        .setPrimaryKeys(internalRow.getString(3).toString())
-                                        .setOption(
-                                                formatOptions(internalRow.getString(4).toString()))
-                                        .setComment(internalRow.getString(5).toString())
-                                        .setUpdateTime(
+                                        .partitionKeys(internalRow.getString(2).toString())
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:82]: Could this cause `NullPointerException` when `internalRow.getString(2)` is null?

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 2]
```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
+                                        .primaryKeys(internalRow.getString(3).toString())
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:83]: Ditto.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 3]
```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
+                                        .option(formatOptions(internalRow.getString(4).toString()))
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:84]: Ditto.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 4]
```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
+                                        .comment(internalRow.getString(5).toString())
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:85]: Ditto.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 5]
```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
+                                        .updateTime(
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:86]: Ditto.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 6]
```diff
@@ -134,9 +133,9 @@ public List<ManifestsVO> getManifest(MetadataDTO dto) {
                     internalRow -> {
                         ManifestsVO manifestsVo =
                                 ManifestsVO.builder()
-                                        .setFileName(internalRow.getString(0).toString())
-                                        .setFileSize(internalRow.getLong(1))
-                                        .setNumAddedFiles(internalRow.getLong(2))
+                                        .fileName(internalRow.getString(0).toString())
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:136]: Ditto.

**Comment by s7monk** [comment]
Thanks for your review, I have updated this PR. @SteNicholas PTAL.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java [hunk 7]
```diff
@@ -71,21 +71,39 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
...
                                                                                 MetadataFieldsModel>>() {}))
-                                        .setPartitionKeys(internalRow.getString(2).toString())
-                                        .setPrimaryKeys(internalRow.getString(3).toString())
-                                        .setOption(
-                                                formatOptions(internalRow.getString(4).toString()))
-                                        .setComment(internalRow.getString(5).toString())
-                                        .setUpdateTime(
-                                                internalRow.getTimestamp(6, 3).toLocalDateTime())
+                                        .partitionKeys(
+                                                internalRow.getString(2) != null
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java:83]: Could this introduce a util method to handle?

**Comment by SteNicholas** [comment]
@s7monk, thanks for updates. Please add the tests for null case.
